saos-batch/src/test/java/pl/edu/icm/saos/batch/core/importer/CtJudgmentImportJobTest.java

```diff
@@ -135,6 +156,18 @@ public void ctJudgmentImportProcessJob_IMPORT_NEW() throws Exception {
         assertJudgment_3b42a6299303c65d869c4806fdcdbf7a();
         assertJudgment_04e47013023d2b315b841f99ccb9c290();
         
+        
```

Comment:
You don't check in this test and tests for other court types if the judgment.getTextContent has been set correctly, for example getTextContent().getFilePath and type.